docs/simple-ruby/index.html

```diff
@@ -33,7 +33,7 @@
           },
         },
         noRecTrack: true,
-        wg: "Internationalizatoin Interest Group",
+        wg: "Internationalization Working Group",
```

Comment:
Besides the typo in i18n, which should indeed be fixed, I'm confused about IG vs WG.
The JLReq-TF charter says:
> The Task Force is part of the W3C Internationalization Interest Group. The Task Force will report the results of its activities as a group back to the W3C Internationalization Working Group.

I picked IG based on the first sentence, assuming that we would as some stage pass it on to the WG, but that it would be considered an IG work item at least initially. Am I reading this wrong?

Reply:
Read a bit further down the charter text and you come to 

>Working Drafts and Notes will be published by the i18n WG, and the i18n WG will work with the task force closely to assist with development and review of the documents.

Reply:
I see, makes sense. Sorry for the noise.